.github/workflows/ci.yml

```
@@ -0,0 +1,64 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - /-dev|-feature|-fix/
+
+  pull_request:
+
+env:
+  JRUBY_OPTS: -J-Xmx896M
+
+jobs:
+  ci:
+    runs-on: ubuntu-20.04
+
+    strategy:
+      matrix:
+        java-version: [8]
+        ruby-version: [jruby-9.1.17.0, jruby-9.2.17.0, jruby-head]
```

Ah, already a new JRuby release is out! I think this should be sufficient:

```suggestion
        ruby-version: [jruby-9.1.17.0, jruby-9.2.18.0, jruby-head]
```

---

It doesn't seem yet available on https://github.com/ruby/setup-ruby though. https://github.com/ruby/setup-ruby/pull/188 was merged but not yet released.

---

Thanks, that clarifies it. We can wait.

---

It's released now